thunder.install

```diff
@@ -186,3 +186,52 @@ function thunder_update_8306() {
   // Output logged messages to related channel of update execution.
   return $updateHelper->logger()->output();
 }
+
+/**
+ * Change entity browser in pararaphs integration.
+ */
+function thunder_update_8307() {
```

Comment:
I think we should add a checklist entry, describe why we do that and how people can do the steps on their own in case we miss something.

Reply:
Should that always fail then?

Reply:
Hm, I think so. I guess otherwise no one will take a look at that...